en/docs/develop/customizations/advanced-ui-customization.md

```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
         npm run build:prod
         ```
 
+### Admin Portal advanced UI customizations 
+
+!!! note "Prerequisites"
+    - **NodeJS** - This is a JavaScript runtime environment required for ReactJS development.
+    - **NPM**
+
+1. Navigate to the `<API-M_HOME>/repository/deployment/server/jaggeryapps/admin` directory in a terminal and run the following command.
+
+     ```js
+     npm ci
+     ```
+
+     This will install the local package dependencies in the Admin applications.
+
+3. Build with customizations
+
+     Run the following command to start the npm build. Note that it will continuously watch for any changes and rebuild the project.
+
+    ```js
+    npm run build:dev
+    ```
+    !!! note "Production deployment"
+        The development build is not optimized and contains a large bundle size. Make sure to use the production build when the customizations are ready for production. Use the following command to get the production-ready build.
+        ```
+        npm run build:prod
+        ```
+3. Make the UI related changes in the respective folder based on the WSO2 API-M Console.
+
+     - If you need to rewrite the admin UI completely, you can make changes in the following directory.
+         - `admin/source`
+     - If you want to override a specific React component or a file from the `source/src/` directory, you need to make the changes in the following directory by only copying the desired file/files.
+         - `admin/override/src`
+
+#### Overriding the API Documentation and Overview components
+
+```sh
+override
+└── src
+    ├── Readme.txt
+    └── app
+        └── components
+            └── Apis
+                └── Details
+                    ├── Documents
+                    │   └── Documentation.jsx
+                    └── Overview.jsx
+```
```

Comment:
Shall we add some explanation here? This has only a file structure so it's unclear what we're asking them to do when you consider the sub heading.